processor/transaction/trace.go

```diff
@@ -40,7 +42,7 @@ func (t *Trace) Transform(transactionId string) common.MapStr {
 	return tr
 }
 
-func (t *Trace) Mappings(pa *payload, tx Event) (string, []m.DocMapping) {
+func (t *Trace) Mappings(pa *payload, tx Event) (time.Time, []m.DocMapping) {
```

Comment:
shouldn't we still return a `string` that we parse out of the `time.Time` object with the proper `time.RFC3339Nano` format that we agreed on?

Reply:
When you have a `time.Time` in a struct that you unmarshal into, Golang automatically converts `string` to `RFC3339Nano` already. No need to do it ourselves.

https://play.golang.org/p/ShKw8ZKqAC